src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -2988,6 +2992,11 @@ private function pushChangesToStagingArea($id) {
     $base_tag = "refs/tags/{$prefix}/base/{$id}";
```

Comment:
Are you sure you want to leave this dead code?

Reply:
This is intentional and recommended by @Pawka - to leave existing code as is to help with merging (so that we're not changing existing code, but instead are just adding new code)